guides/common/modules/proc_adding-life-cycle-environments.adoc

```diff
@@ -23,12 +23,7 @@ To add a life cycle environment to {SmartProxyServer}, complete the following st
 . To synchronize the content on the {SmartProxy}, click the *Overview* tab and click *Synchronize*.
 . Select either *Optimized Sync* or *Complete Sync*.
 +
-ifeval::["{context}" == "{smart-proxy-context}"]
-For definitions of each synchronization type, see {ContentManagementDocURL}Importing_Red_Hat_Content-Recovering_a_Repository[Recovering a Repository] in the _Content Management Guide_.
-endif::[]
-ifeval::["{context}" == "content-management"]
-For definitions of each synchronization type, see xref:Importing_Red_Hat_Content-Recovering_a_Repository[].
-endif::[]
+For definitions of each synchronization type, see {ContentManagementDocURL}Importing_Red_Hat_Content-Recovering_a_Repository[Recovering a Repository].
```

Comment:
This link does not work for me on a Foreman build but works on a Satellite build...

Reply:
My best guess would be that it's a link to a section that we hide in the Foreman build, but that doesn't make sense if the xref worked fine before I changed it to a link. I'll investigate.

Reply:
Ah, the answer is that the xref was broken before! The ID changed from Importing_Red_Hat_Content to Importing_Content and there are a few broken xrefs as a result. I'll fix them here.